static/js/factories/search.js

```diff
@@ -0,0 +1,57 @@
+// @flow
+import casual from "casual-browserify"
+
+import type {
+  CommentResult,
+  PostResult,
+  ProfileResult
+} from "../flow/searchTypes"
+
+export const makeProfileResult = (): ProfileResult => ({
+  author_avatar_medium: casual.url,
+  author_avatar_small:  casual.url,
+  author_bio:           casual.sentence,
+  author_headline:      casual.sentence,
+  author_id:            casual.word,
+  author_name:          casual.full_name,
+  object_type:          "profile"
```

Comment:
Minor nitpick, maybe create constants for the `object_type` and use those to avoid potential spelling errors in the code? 
```javascript
export const OBJECT_TYPE_POST: "post" = "post"
export const OBJECT_TYPE_COMMENT: "comment" = "comment"
export const OBJECT_TYPE_PROFILE: "profile" = "profile"

export type ObjectType =
  | typeof OBJECT_TYPE_POST
  | typeof OBJECT_TYPE_COMMENT
  | typeof OBJECT_TYPE_PROFILE
```

```javascript
object_type:       OBJECT_TYPE_PROFILE
```

Reply:
Sounds good, I updated the types in `searchTypes.js` which will enforce the spelling of the strings here